### PR TITLE
bug 1826095: make schema valid for clusterlogging

### PR DESCRIPTION
**Pull request by jcantrill**
Update CLO CRD to be a valid schema

### CODE REVIEW

File: hack/testing-olm/assertions [hunk 1]
```diff
@@ -14,3 +11,18 @@ assert_resources_exist(){
...
+
+assert_kibana_instance_exists() {
+	# verify kibana crd
+	try_until_success "oc -n $NAMESPACE get kibana instance" ${TIMEOUT_MIN}
+}
+
+is_crd_installed() {
+  crd_name=$1
+  cmd="oc get crd $crd_name"
+  if $cmd; then
```
Comment (syedriko) [inline hack/testing-olm/assertions:23]:
```
$cmd
return $?
```

File: hack/testing-olm/utils [hunk 2]
```diff
@@ -193,3 +193,44 @@ EOL
...
 }
+
+get_github_directory() {
+    local outputdir=$1
+    local ghdir=$2
+    local comp=$3
+    local repo=${4:-openshift}
+    local branch=${5:-master}
+    local name
+    local type
```
Comment (syedriko) [inline hack/testing-olm/utils:204]: type is a bash built-in, can we use another name please?

Comment (jcantrill) [reply hack/testing-olm/utils]: fixed.  this was added from whence it was already commited

**Pull request merged**